src/BloomExe/Edit/EditingView.cs

```diff
@@ -763,6 +763,9 @@ void SaveChangedImage(GeckoHtmlElement imageElement, PalasoImage imageInfo, stri
 			{
 				if (ShouldBailOutBecauseUserAgreedNotToUseJpeg(imageInfo))
 					return;
+				// See https://silbloom.myjetbrains.com/youtrack/issue/BL-2776.
+				if (IsSameFile(imageElement, imageInfo))
```

Comment:
My original code was not preserving cropping.  The revised code in the method does, while trying to avoid unnecessary duplication of files.